roles/openshift_manage_node/tasks/config.yml

```diff
@@ -23,5 +23,16 @@
   delegate_to: "{{ openshift_master_host }}"
   vars:
     l_node_labels: "{{ openshift_node_labels | default({}) }}"
-    l_master_labels: "{{ ('oo_masters_to_config' in group_names) | ternary(openshift_master_node_labels, {}) }}"
+    l_master_labels: "{{ openshift_manage_node_is_master | ternary(openshift_master_node_labels, {}) }}"
     l_all_labels: "{{ l_node_labels | combine(l_master_labels) }}"
+
+- name: Taint master nodes
+  shell: >
+    {{ openshift_client_binary }} adm taint nodes {{ openshift.node.nodename }} {{ item.label }}:{{ item.effect }} --overwrite
+  with_items:
+    - { effect: "NoExecute", label: "{{ l_openshift_master_webconsole_taint_noexecute_labels }}" }
+    - { effect: "NoSchedule", label: "{{ l_openshift_master_webconsole_taint_noschedule_labels }}" }
```

Comment:
I think we missed the file where these are defined?

Reply:
These are static - see https://docs.openshift.com/container-platform/3.7/admin_guide/scheduling/taints_tolerations.html

Reply:
What about `l_openshift_master_webconsole_taint_noexecute_labels` and `l_openshift_master_webconsole_taint_noexecute_labels` though? I don't see them elsewhere in this PR nor in master today?

Reply:
Ah, right, somehow these lines were not included. Fixed